contract-map.json

```diff
@@ -1,6 +1,181 @@
+    "decimals": 8
+  },
+  "0xB9e7F8568e08d5659f5D29C4997173d84CdF2607": {
+    "name": "Swarm City Token",
+    "logo": "swt.jpg",
+    "erc20": true,
+    "symbol": "SWT",
+    "decimals": 18
+  },
+  "0xCb94be6f13A1182E4A4B6140cb7bf2025d28e41B": {
+    "name": "Trustcoin",
+    "logo": "",
+    "erc20": true,
+    "symbol": "TRST",
+    "decimals": 6
+  },
+  "0xBEB9eF514a379B997e0798FDcC901Ee474B6D9A1": {
+    "name": "Melon Token",
+    "logo": "melonport-1.svg",
+    "erc20": true,
+    "symbol": "MLN",
+    "decimals": 18
+  },
+  "0x960b236A07cf122663c4303350609A66A7B288C0": {
+    "name": "Aragon Network Token",
+    "logo": "aragon_isotype.svg",
+    "erc20": true,
+    "symbol": "ANT",
+    "decimals": 18
+  },
+  "0x667088b212ce3d06a1b553a7221E1fD19000d9aF": {
+    "name": "WINGS",
+    "logo": "wings_logo.svg",
+    "erc20": true,
+    "symbol": "WINGS",
+    "decimals": 18
+  },
+  "0xAf30D2a7E90d7DC361c8C4585e9BB7D2F6f15bc7": {
+    "name": "FirstBlood Token",
+    "logo": "1st.svg",
+    "erc20": true,
+    "symbol": "1ST",
+    "decimals": 18
+  },
+  "0x607F4C5BB672230e8672085532f7e901544a7375": {
+    "name": "iEx.ec Network Token",
+    "logo": "rlc.png",
+    "erc20": true,
+    "symbol": "RLC",
+    "decimals": 9
+  },
+  "0xaeC2E87E0A235266D9C5ADc9DEb4b2E29b54D009": {
+    "name": "SingularDTV",
+    "logo": "singulardtv.svg",
+    "erc20": true,
+    "symbol": "SNGLS",
+    "decimals": 0
+  },
+  "0x888666CA69E0f178DED6D75b5726Cee99A87D698": {
+    "name": "ICONOMI",
+    "logo": "iconomi.png",
+    "erc20": true,
+    "symbol": "ICN",
+    "decimals": 18
+  },
+  "0xE0B7927c4aF23765Cb51314A0E0521A9645F0E2A": {
+    "name": "Digix",
+    "logo": "DGD.png",
+    "erc20": true,
+    "symbol": "DGD",
+    "decimals": 0
+  },
+  "0xC66eA802717bFb9833400264Dd12c2bCeAa34a6d": {
+    "name": "Maker",
+    "logo": "logo-maker-4.svg",
+    "erc20": true,
+    "symbol": "MKR",
+    "decimals": 0
+  },
+  "0x0D8775F648430679A709E98d2b0Cb6250d2887EF": {
+    "name": "Basic Attention Token",
+    "logo": "BAT_icon.svg",
+    "erc20": true,
+    "symbol": "BAT",
+    "decimals": 18
+  },
+  "0x6810e776880C02933D47DB1b9fc05908e5386b96": {
+    "name": "Gnosis Token",
+    "logo": "gnosis.svg",
+    "erc20": true,
+    "symbol": "GNO",
+    "decimals": 18
+  },
+  "0xD8912C10681D8B21Fd3742244f44658dBA12264E": {
+    "name": "Pluton",
+    "logo": "plutus-god.svg",
+    "erc20": true,
+    "symbol": "PLU",
+    "decimals": 18
+  },
+  "0x48c80F1f4D53D5951e5D5438B54Cba84f29F32a5": {
+    "name": "Reputation",
+    "logo": "augur_logo.png",
+    "erc20": true,
+    "symbol": "REP",
+    "decimals": 18
+  },
+  "0xa74476443119A942dE498590Fe1f2454d7D4aC0d": {
+    "name": "Golem Network Token",
+    "logo": "golem.svg",
+    "erc20": true,
+    "symbol": "GNT",
+    "decimals": 18
+  },
   "0x6090A6e47849629b7245Dfa1Ca21D94cd15878Ef": {
-    "logo": "ens.svg",
-    "name": "ENS Registrar"
+    "name": "ENS Registrar",
+    "logo": "ens.svg"
```

Comment:
The README says to not touch unmodified entries, but I'm letting it slide b/c it's a mega PR.